rs/consensus/src/consensus/validator.rs

```diff
@@ -3849,13 +3847,14 @@ pub mod test {
 
             let block = pool.make_next_block_with_rank(Rank(1));
             let mut second_block = block.clone();
-            second_block.content.as_mut().context.time += Duration::from_nanos(1);
+            second_block.content.as_mut().context.time += INITIAL_NOTARY_DELAY;
```

Comment:
The purpose of the second and third block is to create an equivocation proof. I modified the timestamp to that the blocks produce different hashes. 

But they still belong to the same rank. I think adding `INITIAL_NOTARY_DELAY` here would be confusing, because the actual value is irrelevant in this case. We just need a way to create three similar blocks of the same rank.

If you want, we can add a comment explaining that.

Reply:
Sorry your first suggestion was very good. I updated the code with it.